feat(lightcurve): Changed schema and added translation functions.

ASHuenchuleo opened the pull request:
## Summary of the changes

Added translation functions to convert from the ZTF schema to the new multistream schema.
Edited the mongo get detection function to return an object with the new schema.
Closes #183 

## Observations

The usual separation between requests for ATLAS and ZTF data is kept.


## If the change is BREAKING, please give more details about the breaking changes

The API schema completely changes, and is now consistent for both surveys.

#### Components that need updates and steps to follow

See #184 


## About this PR:

- [x] You added the necessary documentation for the team to understand your work.
- [x] You linked this PR to the corresponding issue or user story.
- [x] Your changes fulfill the Definition of Done of the issue or user story.
- [ ] Your changes were tested with manual testing before being submitted.
- [x] Your changes were tested with automatic unit tests.
- [x] Your changes were tested with automatic integration tests.

ASHuenchuleo committed:
feat(lightcurve): Changed schema and added translation functions.
Added translation functions to convert from the ZTF schema to the
new multistream schema.
Edited the mongo get detection function to return an object with
the new schema.

ASHuenchuleo committed:
chore(lightcurve): Added docstring

ASHuenchuleo committed:
chore(lightcurve): Linting

dirodriguezm reviewed (commented):
Yo lo veo bien. No sé qué pasó con los tests fallidos, pero encuentro que está bien el update.

HectorxH committed:
fix: ignore fields starting with '_'

This should prevent internal methods and properties added by psql to the
rows to end up in `extra_fields`. Also added a corresponding test.

HectorxH committed:
style: formatting with black

pgallardor reviewed (approved):
Yo lo veo bien igual, asi que le pueden dar a merge
